android/app/build.gradle

```diff
@@ -132,8 +129,6 @@ dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
-    compile project(':react-native-image-picker')
```

Comment:
Yeah, the native parts are really edit-at-your-own-risk. Not sure if it's a flaw in the tooling or we are not extending it correctly.